src/models/elap1-r1/README.md

```diff
@@ -0,0 +1,44 @@
+# Eland Pro 1U (elap1-r1)
+
+![Eland Pro 1U](./img/elap1-r1-frontports.webp)
+
+The System76 Eland Pro 1U is a rack-mount server with the following specifications:
+
+- CPU options
+  - Supports AMD EPYC™ 7003 series processor family
+  - Single processor
+- Memory
+  - Up to 8192GB (32x256GB) ECC DDR4 @ 3200 MHz
+- Storage
+  - 8 x 2.5" SATA/SAS and 2 x 2.5" NVMe hot-swappable HDD/SSD bays
+  - 1 x PCIe Gen3 M.2 slot
+  - Note: CPU TDP is limited to 225w if using M.2 device
+- Networking
+  - 2x 1GbE LAN ports ([1 x Intel® I350-AM2](https://ark.intel.com/content/www/us/en/ark/products/52968/intel-ethernet-controller-i350am2.html))
+  - 1x 10/100/1000 management LAN
+- Expansion
+  - 1x PCIe 4.0 x16
+- Front Overview:
+  - 1 x USB 3.0
+  - 1 x Power button with LED
+  - 1 x ID button with LED
+  - 1 x NMI button
+  - 1 x Reset button
+  - 2 x LAN activity LEDs
+  - 1 x HDD activity LED
+  - 1 x System status LED
+- Back Overview:
+  - 3 x USB 3.0
+  - 1 x VGA
+  - 2 x RJ45
+  - 1 x [MLAN](./elap1-r1_manual.pdf#page=143)
+  - 1 x ID button with LED
+- Power Supply
+  - 2 x 1200W redundant PSUs, 80 PLUS Platinum
+  - AC Input: 100-240Vac/ 12-7A, 50-60Hz
+- Dimensions
+  - 43.8cm × 73.0cm × 4.35cm
+- Weight
+  - 20.32 kg
+- Model
+  - elap1-r1 - [Gigabyte R182-Z9{0,1,2}](./elap1-r1_manual.pdf)
```

Comment:
The storage listed on this page (8x SATA/SAS and 2x NVMe)  is only for the R182-Z91. The Z92 has 10 NVMe, and the Z90 has 4x SATA/SAS.

@leviport Do you know if the elap1-r1 actually had the option of all three of these chassis, or if it was just the Z91?

Reply:
I don't recall, sorry.

Reply:
From [blazer search](https://blazer.origin76.com/queries/480-look-up-server-chassis?model=elap1-r1), the `elap1-r1` only shipped with a single chassis: Gigabyte R182-Z91 1U 7003 Dual Socket